man/systemd-system.conf.xml

```diff
@@ -525,6 +525,18 @@
         details. Note that this setting has no effect on the OOM score adjustment value of the service
         manager process itself, it retains the original value set during its invocation.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>DefaultSmackProcessLabel=</varname></term>
+
+        <listitem><para>Takes a <option>SMACK64</option> security label as the argument. The process executed
+        by a unit will be started under this label if <varname>SmackProcessLabel=</varname> is not set in the
+        unit. See <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        for the details.</para>
+
+        <para>If the value is <literal>/</literal> only labels labels specified with <varname>SmackProcessLabel=</para>
+        are assigned and the compile-time default is ignored.</listitem>
```

Comment:
is this common in SMACK land, to use `/` as marker for "unset"? or did you invent this here?

Reply:
This is my invention. According to SMACK [documentation](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/admin-guide/LSM/Smack.rst?id=e8a4e1c1bb697b1d9fc48f0e56dc0f50bc024bee#n438) 

> […] Single character labels using special characters, that being anything other than a letter or digit, are reserved for use by the Smack development team.

but 

> […] Smack labels cannot contain unprintable characters, the "/" (slash), the "\" (backslash), the "'" (quote) and '"' (double-quote) characters. Smack labels cannot begin with a '-'. This is reserved for special options. […]

Which means we can be sure `/` won't be used anywhere in SMACK.

> Every task on a Smack system is assigned a label. The Smack label
of a process will usually be assigned by the system initialization
mechanism.

Therefore, there is no notion of _unset_ in SMACK. This is going to by purely systemd logic, to use kernel default instead of the compiled in systemd default.

Reply:
but "-" sounds like a safer option then, no?

Reply:
The documentation says.

> Smack labels cannot begin with a '-'. This is reserved for special options.

I'd rather not use something that is marked as reserved.